DigitalLearningSolutions.Web/Views/Shared/SearchablePage/Configurations/_ResultsCountAndItemsPerPage.cshtml

```diff
@@ -0,0 +1,9 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.Common.SearchablePage
+@model BaseSearchablePageViewModel
+
+  <div class="nhsuk-u-margin-top-4 items-per-page-box-container">
+    <div class="nhsuk-u-margin-right-auto nhsuk-u-margin-bottom-3">

```

Comment:
HEE asked in the show and tell about changing some of these margins

Reply:
Sorted, gap is now gone.
Desktop:
![image](https://user-images.githubusercontent.com/18046982/148920970-660d29c2-4531-4446-a35d-dd64938e96d8.png)
Mobile:
![image](https://user-images.githubusercontent.com/18046982/148921007-696753f0-829a-4d94-8422-77a4eeca7896.png)